_posts/fr/2020-08-12-meilleure-technologie-grace-au-micro-approvisionnement-partie-1.md

```diff
@@ -0,0 +1,120 @@
+
+***
+
+Le gouvernement du Canada (GC) achète beaucoup de biens et de services.
+En 2016, le GC a dépensé au total environ 16 milliards de dollars en biens et services.
+Du point de vue de la TI, ces dépenses peuvent comprendre des articles coûteux comme de nouveaux centres de données et de nouvelles applications nationales qui coûtent des dizaines de millions de dollars, mais également des articles de faible valeur, comme le fait de payer quelqu'un pour construire un petit prototype en quelques jours, ou d'embaucher un consultant pour réaliser une analyse de l'écart.
+
+Afin de s'assurer que l'approvisionnement fonctionne bien à l'échelle du GC, les équipes d'approvisionnement travaillent sans relâche aux efforts de modernisation, y compris à l'élaboration d'un [système d'approvisionnement électronique](https://achatsetventes.gc.ca/initiatives-et-programmes/la-solution-d-achats-electroniques/a-propos), pour la mise en œuvre [d'un approvisionnement agile](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/appagile-proagile-fra.html), [des projets pilotes sur l'approvisionnement social](https://achatsetventes.gc.ca/donnees-sur-l-approvisionnement/appels-d-offres/PW-18-00828087) et [la simplification du libellé des contrats](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/simpcont-simpcont-fra.html).
+Fait intéressant, les approvisionnements de faible valeur ne semblent pas figurer dans la liste des priorités de modernisation des approvisionnements.
+Cela pourrait être dû au fait que ces approvisionnements présentent un faible risque et ne font généralement pas les manchettes.
+Toutefois, pour la même raison, ils offrent l'occasion d'acquérir et d'expérimenter des produits de TI.
+
+<p class="tab">
+Imaginez que vous gérez un produit de TI et qu'un membre de l'équipe remarque un bogue dans l'une des composantes de source ouverte, mais n'a pas le temps de le régler.
+Que se passerait-il si vous pouviez afficher une occasion de corriger ce bogue pour 2 000 $, choisir une personne pour faire le travail et recevoir le code corrigé et testé dans un délai de quelques jours ?
+</p>
+
+Pour le GC, les marchés de faible valeur sont définis dans le [Guide des approvisionnements](https://achatsetventes.gc.ca/politiques-et-lignes-directrices/guide-des-approvisionnements/section/3/60) de SPAC comme des marchés dont la valeur est de 25 000 $ ou moins pour des biens, ou de 40 000 $ ou moins pour des services.
+La majorité du budget de dépenses en biens et services au GC porte sur des articles de faible valeur.
```

Comment:
As per the English version, this sentence seems to indicate that the majority of money spent is on LDV whereas it should be the majority of awarded contracts (not $ value)